### PR TITLE
Implemented HelloWorldProvider

### PR DESCRIPTION
**Pull request by osr-pafoca-test**
idea is to provide the hello world string

### CODE REVIEW

File: HelloWorldProvider.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+public class HelloWorldProvider{
+  
+  private static final String HELLO = "Hello";
+  private static final String WORLD = "World";
```
Comment (osr-reviewcrawler-01) [inline HelloWorldProvider.java:4]: missing "!"

**Comment by osr-reviewcrawler-01** [comment]
dont like this idea

**Comment by osr-pafoca-test** [comment]
but i like it

**Comment by osr-reviewcrawler-01** [comment]
ok ok

**Comment by osr-pafoca-test** [comment]
This is fine

**Pull request merged**